src/sas/webfit/settings.py

```diff
@@ -0,0 +1,322 @@
+"""
+Django settings for webfit project.
+
+Generated by 'django-admin startproject' using Django 4.2.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+import os
+from pathlib import Path
+from rest_framework import permissions, versioning
+from django.core.signals import setting_changed
+
+from django.conf import settings
+# Import from `django.core.signals` instead of the official location
+# `django.test.signals` to avoid importing the test module unnecessarily.
+from django.core.signals import setting_changed
+from django.utils.module_loading import import_string
+from rest_framework import ISO_8601
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent
+
+#imports for knox
+from datetime import timedelta
+from rest_framework.settings import api_settings
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
```

Comment:
If I understand correctly a local instance of django server will be initiated while running API. I guess SECRECT_KEY should be set up by user (and not distributed).